src/components/Boards/Boards.tsx

```diff
@@ -1,22 +1,29 @@
 import React from 'react';
-import useGroups from '../../lib/hooks/useGroups';
-
-export default function Boards() {
-  const { data, isLoading, error } = useGroups();
-
-  if (isLoading) {
-    return <div>Loading...</div>;
-  }
-
-  if (error) {
-    return <div>Error Loading Boards</div>;
-  }
+import Paper from '@material-ui/core/Paper';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListAltIcon from '@material-ui/icons/ListAlt';
 
+export default function(props: {
```

Comment:
Create interface instead of inline?

Reply:
Do we want to use type or interface here?

Reply:
I'm okay with either. I generally prefer `interface`